src/test/java/org/assertj/core/api/localdate/LocalDateAssert_isEqualTo_Test.java

```diff
@@ -39,6 +39,12 @@ public void test_isEqualTo_assertion(LocalDate referenceDate) {
     verify_that_isEqualTo_assertion_fails_and_throws_AssertionError(referenceDate);
   }
 
+  @Test
+  public void test_isEqualTo_assertion_with_XmlGregorianCalendar() {
+    assertThat(localDate1).isEqualTo(xmlOfLocalDate1);
+    assertThat(localDate1).isNotEqualTo(xmlOfLocalDate2);
```

Comment:
`isNotEqualTo` assertion will always success whatever `XmlGregorianCalendar` is given (even xmlOfLocalDate1!) since it does not any conversion to LocalDate to `XmlGregorianCalendar`.
To be consistent, we need to add `isNotEqualTo (XMLGregorianCalendar xmlGregorianCalendar)`